src/assets/usage-examples-code/input/key_typed/key_typed-1-type-key.cpp

```diff
@@ -0,0 +1,41 @@
+/**
+ * Usage Example: creating a sprite
+ **/
+
+#include "splashkit.h"
+
+int main()
+{
+    // widow to draw the sprite on
+    window start = open_window("create_sprite", 600, 600);
+
+    // bitmap for creating a sprite
+    bitmap player = load_bitmap("playerBmp", "protSpriteSheet.png");
+    bitmap_set_cell_details(player, 31, 32, 4, 3, 12);
+
+    // creating the player sprite
+    sprite player_sprite = create_sprite(player);
+
+    // setting the co-ordinates in refrence to the window
```

Comment:
Isn't this centering the sprite on the window, rather then setting it's coordinates relative to?